dashboards/ec2/ec2_application_load_balancer_detail.sp

```diff
@@ -0,0 +1,545 @@
+dashboard "ec2_application_load_balancer_detail" {
+  title         = "AWS EC2 Application Load Balancer Detail"
+  documentation = file("./dashboards/ec2/docs/ec2_application_load_balancer_detail.md")
+
+  tags = merge(local.ec2_common_tags, {
+    type = "Detail"
+  })
+
+  input "alb" {
+    title = "Select an Application Load balancer:"
+    query = query.ec2_application_load_balancer_input
+    width = 4
+  }
+
+  container {
+
+    card {
+      width = 2
+      query = query.ec2_application_load_balancer_state
+      args  = [self.input.alb.value]
+    }
+
+    card {
+      width = 2
+      query = query.ec2_application_load_balancer_scheme
+      args  = [self.input.alb.value]
+    }
+
+    card {
+      width = 2
+      query = query.ec2_application_load_balancer_ip_type
+      args  = [self.input.alb.value]
+    }
+
+    card {
+      width = 2
+      query = query.ec2_application_load_balancer_az_zone
+      args  = [self.input.alb.value]
+    }
+
+    card {
+      width = 2
+      query = query.ec2_application_load_balancer_logging_enabled
+      args  = [self.input.alb.value]
+    }
+
+    card {
+      width = 2
+      query = query.ec2_application_load_balancer_deletion_protection
+      args  = [self.input.alb.value]
+    }
+
+  }
+
+  with "acm_certificates_for_ec2_application_load_balancer" {
+    query = query.acm_certificates_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "cloudfront_distributions_for_ec2_application_load_balancer" {
+    query = query.cloudfront_distributions_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "ec2_instances_for_ec2_application_load_balancer" {
+    query = query.ec2_instances_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "ec2_load_balancer_listeners_for_ec2_application_load_balancer" {
+    query = query.ec2_load_balancer_listeners_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "ec2_target_groups_for_ec2_application_load_balancer" {
+    query = query.ec2_target_groups_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "s3_buckets_for_ec2_application_load_balancer" {
+    query = query.s3_buckets_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "vpc_security_groups_for_ec2_application_load_balancer" {
+    query = query.vpc_security_groups_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "vpc_subnets_for_ec2_application_load_balancer" {
+    query = query.vpc_subnets_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  with "vpc_vpcs_for_ec2_application_load_balancer" {
+    query = query.vpc_vpcs_for_ec2_application_load_balancer
+    args  = [self.input.alb.value]
+  }
+
+  container {
+    graph {
+      title = "Relationships"
+      type  = "graph"
```

Comment:
Don't we need the `direction` parameter here?

Reply:
Default is TD